experiment_handling.py

```diff
@@ -95,17 +95,17 @@ def slave(slavecalcs, sid):
 
 
 #TODO: needs more efficient resave handling for every large runs / experiments
-def resave_data(save_path, parameter_combinations, index, eva, name,
-                sample_size=None, badmisskey=None):
+def resave_data(source_path, parameter_combinations, index, eva, name,
+                sample_size=None, badmisskey=None, save_path="./data"):
```

Comment:
The default value for the save_path variable is given in the function definition